src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,30 +1,82 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
-
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
-
-      <span className="icon is-right" style={{ pointerEvents: 'all' }}>
-        {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-        <button data-cy="clearSearchButton" type="button" className="delete" />
-      </span>
-    </p>
-  </form>
-);
+import { ChangeEvent, useState } from 'react';
+import { StatusTodo } from '../../types/StatusTodo';
+
+type Props = {
+  selectedStatus: StatusTodo;
+  setSelectedStatus: (status: StatusTodo) => void;
+  setQuery: (query: string) => void;
+};
+
+export const TodoFilter: React.FC<Props> = ({
+  selectedStatus,
+  setSelectedStatus,
+  setQuery,
+}: Props) => {
+  const [query, setSelectedQuery] = useState('');
```

Comment:
```suggestion
  const [selectedQuery, setSelectedQuery] = useState('');
```
1.state should name the same as set function
2. why you need 2 query states??you already has state in App?